content/help/ops/setup/webhook/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Dynamic Admission Webhooks Overview
+description: Provides a general overview of Istio's use of Kubernetes webhooks and the related issues that can arise.
+weight: 10
+---
+
+From [Kubernetes mutating and validating webhook mechanisms](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/):
+
+> Admission webhooks are HTTP callbacks that receive admission requests
+and do something with them. You can define two types of admission
+webhooks, validating admission Webhook and mutating admission
+webhook. With validating admission Webhooks, you may reject requests
+to enforce custom admission policies. With mutating admission
+Webhooks, you may change requests to enforce custom defaults.
+
+The webhook setup guides assuming general familiarity with Kubernetes
+Dynamic Admission Webhooks. Consult the [Kubernetes API references](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/)) for
+detailed documentation of the mutating and validating webhook configuration.
+
+## Prerequisites
+
+See the [quick start prerequisites](https://istio.io/docs/setup/kubernetes/quick-start/#prerequisites)
+for Kubernetes provider specific setup instructions. Webhooks will not
+function properly if the cluster is misconfigured. You can follow
+these steps once the cluster has been configured and dynamic
+webhooks and dependent features are not functioning properly.
+
```

Comment:
Can we say something about the expected observed behavior if webhooks are in fact not working? That is, how would an operator know they have a problem with webhooks?

Reply:
And similarly, how can the operator validate that everything is working correctly?

Reply:
Problems manifest in different ways depending on the webhook. I've called out specific observable problems in the validation and injection pages.